dev-db/sqlitebrowser/additional-files/sqlitebrowser.rdef

```diff
@@ -0,0 +1,64 @@
+
+resource app_flags B_SINGLE_LAUNCH | B_ARGV_ONLY;
+
+resource app_version {
+	major  = 3,
+	middle = 9,
+	minor  = 1,
```

Comment:
Although useing jhard-coded values in the rdef file is perfectly correct, we often rename the rdef file to rdef.in and use this:
```
	major  = @MAJOR@,
	middle = @MIDDLE@,
	minor  = @MINOR@,
```
and later use sed to produce a rdef from the rdef.in.
I would recommend doing so for this recipe.
This has the advantage of not having to updated the rdef file (or the rdef.in file, since we rename it to make it clean that it has patterns.)